tools/make-ext4fs/Makefile

```diff
@@ -10,13 +10,13 @@ include $(TOPDIR)/rules.mk
 PKG_NAME:=make-ext4fs
 PKG_VERSION:=2016-08-14
 
-PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.xz
 PKG_SOURCE_URL=$(LEDE_GIT)/project/make_ext4fs.git
 PKG_SOURCE_PROTO:=git
 PKG_SOURCE_VERSION:=484903e4332be6c317f531b008cb2a841a18c506
 PKG_SOURCE_SUBDIR:=$(PKG_NAME)-$(PKG_VERSION)
-PKG_MIRROR_MD5SUM:=b80d6ddcd6f6925e2f5b80775bb96f3e
-PKG_CAT:=zcat
+PKG_MIRROR_MD5SUM:=
+# PKG_CAT:=zcat
```

Comment:
Couldn't find any documentation or figure out what it actually did, there are more occurrences of stray lines in package Makefiles so I'd rather do a tree-wide cleanup afterwards.